src/main/resources/application.yml

```diff
@@ -50,7 +50,7 @@ server:
 exam-results-transmitter-service:
   session-url: http://localhost:32842
   assessment-url: http://localhost:32841
-  exam-url: http://localhost:8081
+  exam-url: http://localhost:80
```

Comment:
No, exam runs on port 80 in the docker compose environment.  I felt like we should be able to stand this up and run this against the exam within compose rather than forcing us to run exam locally as well